src/gui/options.ui

```diff
@@ -2290,6 +2290,37 @@
              </widget>
             </item>
             <item>
+             <widget class="QGroupBox" name="checkEnableAddTrackers">
+              <property name="sizePolicy">
+               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+                <horstretch>0</horstretch>
+                <verstretch>0</verstretch>
+               </sizepolicy>
+              </property>
+              <property name="title">
+               <string>A&amp;utomatically add these trackers to new downloads:</string>
```

Comment:
I'm thinking in changing the text to "Automatically add trackers to new downloads:"